ci/terraform/oidc/token.tf

```diff
@@ -16,6 +16,9 @@ module "oidc_token_role" {
     local.user_credentials_encryption_policy_arn,
     local.user_profile_encryption_policy_arn
   ]
+  extra_tags = {
+    Service = "token"
```

Comment:
Should this be `oidc-token` see `auth-token` previously defined.  Similar comment about the following names that could apply to several of the apis.

Reply:
same as before: because module tag = endpoint_name, role tag has to be the same. Pulled out to variable here too.